storage/mysql/log_storage_test.go

```diff
@@ -371,6 +371,42 @@ func TestQueueLeaves(t *testing.T) {
 	}
 }
 
+func TestQueueLeavesDuplicateBigBatch(t *testing.T) {
+	t.Skip("Known Issue: https://github.com/google/trillian/issues/1845")
```

Comment:
Is there a way to skip these conditionally, depending on the environment?

Reply:
It's possible but more work. It would involve skipping if the DB is not present, then querying it for the version, parsing it, and then skipping based on MariaDB maj=10, min>=3.

I can do that and test it on MariaDB versions, but I've spent enough time on this and would need to test such code works for MySQL and I have little appetite for that.